resources/sass/app.scss

```diff
@@ -262,70 +260,71 @@ dl.import-meta .download-csv {
 
 .button {
     padding: 6px 13px;
-    font-weight: 700;
+    font-weight: $wikit-Button-font-weight;
+    line-height: $wikit-Button-line-height;
     display: inline-block;
-    color: $color-base-10;
-    border-width: 1px;
-    border-style: solid;
-    border-color: $color-base-50;
-    border-radius: 2px;
-    background-color: $color-base-90;
-    transition-duration: 100ms;
+    color: $wikit-Button-normal-neutral-color;
+    border-width: $wikit-Button-border-width;
+    border-style: $wikit-Button-border-style;
+    border-color: $wikit-Button-normal-neutral-border-color;
+    border-radius: $wikit-Button-border-radius;
+    background-color: $wikit-Button-normal-neutral-background-color;
+    transition-duration: $wikit-Button-transition-duration;
 
     &:hover {
         text-decoration: none;
-        color: $color-modifier-lighten-base-10;
-        border-color: $color-base-50;
-        background-color: $color-base-100;
+        color: $wikit-Button-normal-neutral-hover-color;
+        border-color: $wikit-Button-normal-neutral-hover-border-color;
+        background-color: $wikit-Button-normal-neutral-hover-background-color;
         transition-property: background-color, border-color;
```

Comment:
We could use: $wikit-Button-transition-property

Reply:
Done in bba763be328c5e94d3eda0be8acab4717fca42cc.